tf_agents/environments/wrappers.py

```diff
@@ -719,3 +719,45 @@ def _step(self, action):
 
     time_step = self._env.step(action)
     return self._add_history(time_step, action)
+
+
+@gin.configurable
+class MultiDiscreteToDiscreteWrapper(PyEnvironmentBaseWrapper):
+  """Converts an environment with Multidiscrete action space into a Discrete one."""
```

Comment:
Document how the multiple discrete spaces are mapped to a single one. You can show a remapping of the sample action in the multidiscrete space from gym:

https://github.com/openai/gym/blob/master/gym/spaces/multi_discrete.py